metadata-ingestion/src/datahub/sql_parsing/tool_meta_extractor.py

```diff
@@ -100,6 +112,7 @@ def create(
                     looker_user_mapping = from_serialized_value(
```

Comment:
Can you check if SerializedResourceValue.as_raw_json can be used here instead. With platform resource sdk available, module datahub.ingestion.source.common.resource should ideally not be needed.